src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/TeamCreation/CheckIcon.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export const CheckIcon = () => (
+  <svg viewBox="0 0 20 20" fill="inherit" xmlns="http://www.w3.org/2000/svg">
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M10 0C4.47715 0 0 4.47715 0 10C0 15.5228 4.47715 20 10 20C15.5228 20 20 15.5228 20 10C20 4.47715 15.5228 0 10 0ZM17.15 7.33117L8.32107 16.0724L2.78898 10.6157L4.55675 8.84789L8.32107 12.5368L15.3823 5.56341L17.15 7.33117Z"
+    />
+  </svg>
+);
```

Comment:
it seems we already have that icon in the ui-kit, called `CheckRoundIcon`

![image](https://github.com/user-attachments/assets/b92be285-e3a2-47e9-a1a4-df488867dae8)